lib/node_modules/@stdlib/math/base/special/binet/benchmark/c/benchmark.c

```diff
@@ -99,16 +100,19 @@ double binet( double x ) {
 * @return elapsed time in seconds
 */
 static double benchmark( void ) {
+	double x[ 100 ];
 	double elapsed;
 	double t;
-	double x;
 	double y;
 	int i;
 
+	for ( i = 0; i < 100; i++ ) {
+		x[ i ] = floor( 79.0*rand_double() );
+	}
+
 	t = tic();
 	for ( i = 0; i < ITERATIONS; i++ ) {
-		x = floor( 79.0*rand_double() );
-		y = binet( x );
+		y = stdlib_base_binet( x[ i % 100 ] );
```

Comment:
```suggestion
		y = stdlib_base_binet( x[ i%100 ] );
```

Minor change just to be consistent with other refactoring related PRs. This change applies throughout the PR.